PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,29 @@
 # Pull Request
 
+### Priority
+
+- [ ] **Urgent** - Drop everthing this is a major block
+- [ ] **Important** - I can wait a morning/afternoon
+- [ ] **Soon** - In the next couple of days
+- [ ] **Not urgent** - It's chill, sometime in the next week
+  
 ### Pull Request Type:
 
-- [ ] Feature
-- [ ] Release
-- [ ] Hotfix
+- [ ] **Feature**
+- [ ] **Release**
+- [ ] **Hotfix**
 
 ### Pull Request Description:
-
+This should include the following.
+ - A link to the ClickUp ticket
+ - The aim of the PR 
```

Comment:
I think the aim of the PR should be clear from the clickup task.
What I find helpful is for the crux of the ticket or even better - the most technically controversial or challenging part of the ticket is highlighted. e.g. "The solution uses shared memory and semaphores instead of a pipe as was previously discussed because ..."
so maybe a note saying 'if possible highlight' these.